src/zeit/retresco/tests/test_update.py

```diff
@@ -166,3 +167,41 @@ def index(tms, content):
         zeit.cms.workflow.interfaces.IPublishInfo(content).urgent = True
         zeit.cms.workflow.interfaces.IPublish(content).publish()
         zeit.workflow.testing.run_publish()
+
+
+class TMSUpdateRequestTest(zeit.cms.testing.BrowserTestCase):
+
+    layer = zeit.retresco.testing.ZCML_LAYER
+
+    def test_endpoint_avoids_get(self):
+        b = zope.testbrowser.testing.Browser()
+        with self.assertRaisesRegexp(mechanize.HTTPError,
```

Comment:
In anderen Tests fangen wir `urllib2.HTTPError`, das ist glaub etwas "schöner" da abstrakter.